mollie/api/resources/captures.py

```diff
@@ -36,5 +36,5 @@ def __init__(self, client, settlement):
         self._settlement = settlement
         super().__init__(client)
 
-    def get_resource_path(self):
-        return f"settlements/{self._settlement.id}/captures"
+    def get_resource_path(self) -> str:
+        return f"settlements/{self._settlement.id}/captures"  # type:ignore
```

Comment:
Why use `# type: ignore` here?

Reply:
The `self._settlement` property is typed as `None`, so it cannot have an `.id` property. 

Some older python versions (tested with 3.7) cannot parse proper typing for `self._settlement` so we can't add that (yet). Ignoring the message about incorrect typing on `self._settlement` here is about the best we can do.